### ISSUE TITLE
[bug]: Inpainting error : Input should be 'Gaussian Blur', 'Box Blur' or 'Staged'

### ISSUE THREAD

**Issue by Nashar410** [invoke-ai/InvokeAI#6127]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Windows

### GPU vendor

Nvidia (CUDA)

### GPU model

QUADRO TX 6000 (Shadow PC Pro)

### GPU VRAM

20

### Version number

4.0.1

### Browser

Chrome

### What happened

Hello !

At first, thank you for your work !

I'm trying to inpaint (from the unified canva), no matter what I do I get this error in the red box at the bottom left when clicking on "Invoke" :

``Input should be 'Gaussian Blur', 'Box Blur' or 'Staged'
Path: body.batch.graph.nodes.create_gradient_mask.coherence_mode
``

Nothing showing in the console, but there is some info in the network tabs from the chrome dev tool : 

Payload (quite long) : 
[paylaod_invokeai.txt](https://github.com/invoke-ai/InvokeAI/files/14854673/paylaod_invokeai.txt)

Response (parsed in json by chrome dev tool) :
```json
{
    "detail": [
        {
            "type": "literal_error",
            "loc": [
                "body",
                "batch",
                "graph",
                "nodes",
                "create_gradient_mask",
                "coherence_mode"
            ],
            "msg": "Input should be 'Gaussian Blur', 'Box Blur' or 'Staged'",
            "input": "unmasked",
            "ctx": {
                "expected": "'Gaussian Blur', 'Box Blur' or 'Staged'"
            },
            "url": "https://errors.pydantic.dev/2.6/v/literal_error"
        }
    ]
}
```

### What you expected to happen

The standard behavior (being able to inpaint with a mask)

### How to reproduce the problem

I'm inpainting : 
- Select an image
- Reduce the canvas to a selected part (facultative)
- Draw the masks 
- Adding some control (facultative)

The error happen when I have drawn a mask and I hit 'Invoke'.

### Additional context

I upgrade the 3.7 to the 4.01 as soon as it was available. I don't have the time to clean install a fresh version right now, but I will do it as soon as possible.

**Comment by Nashar410** [comment]
Good old Private Browsing solved the problem, so I just cleared all of the website's cache data from all of my browsers, and now everything is fine.

tl;dr:

Clear the site data (cache, etc.) from your browser's dev tools. It even fixed some weird CSS issues I had!

**Comment by RFedz007989** [comment]
Hi,

In 4.0 there's something new Gaussian Blur, Box Blur & Staged. Here before you do Invoke you have select the composition box drop down above controlnet there you need to select which blur you want or staged after doing this you click Invoke you will get output result without this error...

Thank You
